wsrep_api.h

```diff
@@ -111,8 +111,9 @@ typedef void (*wsrep_log_cb_t)(wsrep_log_level_t, const char *);
 #define WSREP_CAP_UNORDERED             ( 1ULL << 12 )
 #define WSREP_CAP_ANNOTATION            ( 1ULL << 13 )
 #define WSREP_CAP_PREORDERED            ( 1ULL << 14 )
-#define WSREP_CAP_SNAPSHOT              ( 1ULL << 15 )
```

Comment:
Can changing the values of SNAPHOT and NBO cause incompatibilities with previous versions of mysql-wsrep and/or galera? If there is no some special reason, then maybe it would be better to leave them as they are and add the new STREAMING as 1<<17?

Reply:
No previous version of Galera ever exposed these flags and they have never been published.
STREAMING capability is more basic and likely will be released first so it makes sense to put it ahead for SNAPSHOT and NBO.